src/apps/across-v2/across.pool.definitions.ts

```diff
@@ -0,0 +1,84 @@
+import { Acrossv2PoolDefinition,  AcrosstTokenDefinition } from './across.types';
+
+export const ACROSS_V2_POOL_DEFINITIONS: Acrossv2PoolDefinition[] = [
+  {
+    queryKey: 'HubPool',
+    label: 'Across HubPool',
+    poolAddress: '0xc186fA914353c44b2E33eBE05f21846F1048bEda',
+    chainId: 1,
+    isSpokePool: false,
+  },
+  {
+    queryKey: 'Ethereum_SpokePool',
+    label: 'Across Ethereum_SpokePool',
+    poolAddress: '0x4D9079Bb4165aeb4084c526a32695dCfd2F77381',
+    chainId: 1,
+    isSpokePool: true,
+  },
+  {
+    queryKey: 'Optimism_SpokePool',
+    label: 'Across Optimism_SpokePool',
+    poolAddress: '0xa420b2d1c0841415A695b81E5B867BCD07Dff8C9',
+    chainId: 10,
+    isSpokePool: true,
+  },
+  {
+    queryKey: 'Polygon_SpokePool',
+    label: 'Across Polygon_SpokePool',
+    poolAddress: '0x69B5c72837769eF1e7C164Abc6515DcFf217F920',
+    chainId: 137,
+    isSpokePool: true,
+  },
+  {
+    queryKey: 'Boba_SpokePool',
+    label: 'Across Boba_SpokePool',
+    poolAddress: '0xBbc6009fEfFc27ce705322832Cb2068F8C1e0A58',
+    chainId: 288,
+    isSpokePool: true,
+  },
+  {
+    queryKey: 'Arbitrum_SpokePool',
+    label: 'Across Arbitrum_SpokePool',
+    poolAddress: '0x02fbb64517E1c6ED69a6FAa3ABf37Db0482f1152',
+    chainId: 42161,
+    isSpokePool: true,
+  },
+];
+
+export const POOL_TOKENS:AcrosstTokenDefinition[] = [
+  {
+    queryKey: 'badger',
+    symbol: 'BADGER',
+    tokenAddress: '0x3472a5a71965499acd81997a54bba8d852c6e53d',
```

Comment:
This won't work... the hub and spoke pools are on different networks, so the addresses of the tokens that can be deposited are different on each network.

You'll need to create a configuration of tokens _per network_.